core/src/main/java/io/aiven/klaw/service/Pager.java

```diff
@@ -0,0 +1,107 @@
+package io.aiven.klaw.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiFunction;
+
+public class Pager {
+  public static final int DEFAULT_REC_PER_PAGE = 20;
+
+  static int deriveCurrentPage(String pageNo, String currentPage, int totalPages) {
+    return switch (pageNo) {
+      case ">" -> Integer.parseInt(currentPage) + 1;
+      case ">>" -> totalPages;
+      case "<" -> Integer.parseInt(currentPage) - 1;
+      case "<<" -> 1;
+      default -> currentPage.isBlank() ? 1 : Integer.parseInt(currentPage);
+    };
+  }
+
+  private static void getAllPagesList(
```

Comment:
this method already exists in commonUtilsService. Can we avoid this duplicate block. either delete here or there.

Reply:
removed from `commonUtilsService`